projects/packages/my-jetpack/_inc/components/product-card/index.jsx

```diff
@@ -34,6 +35,30 @@ const Menu = ( {
 	const showStandaloneOption =
 		hasStandalonePlugin && ( ! isStandaloneInstalled || ! isStandaloneActive );
 
+	if ( productStatus === PRODUCT_STATUSES.ACTIVE ) {
+		items.push( {
+			label: __( 'Manage', 'jetpack-my-jetpack' ),
+			onClick: onManage,
+			icon: external,
+		} );
+	}
+
+	if ( showStandaloneOption && ! isStandaloneInstalled ) {
+		items.push( {
+			label: __( 'Install Plugin', 'jetpack-my-jetpack' ),
+			onClick: onInstall,
+			icon: download,
+		} );
+	}
+
+	if ( showStandaloneOption && isStandaloneInstalled && ! isStandaloneActive ) {
+		items.push( {
+			label: __( 'Activate Plugin', 'jetpack-my-jetpack' ),
+			onClick: onActivate,
+			icon: check,
+		} );
+	}
```

Comment:
I think it is better not to manipulate the prop directly. We're changing the original array, and if we need to rely on it at some moment, it will be another one. And this way, we change the layout, not including the divider between "actions" and "install/activate." If you prefer to keep using array, better to split between custom items and default items, but I vote to keep the default items declarative.

Reply:
Changed the code back to the declarative format and also added a check to show the divider.